common/djangoapps/student/views.py

```diff
@@ -132,6 +132,14 @@
 SETTING_CHANGE_INITIATED = 'edx.user.settings.change_initiated'
 # Used as the name of the user attribute for tracking affiliate registrations
 REGISTRATION_AFFILIATE_ID = 'registration_affiliate_id'
+REGISTRATION_UTM_PARAMETERS = {
```

Comment:
I want to make changing the names of these attributes difficult.  It has a big impact to analytics if these are changed.